codegen/gateway.go

```diff
@@ -947,7 +947,10 @@ func parseMiddlewareConfig(
 		}
 		instanceConfig := filepath.Join(fullMiddlewareDir, file.Name(), "middleware-config.json")
 		bytes, err := ioutil.ReadFile(instanceConfig)
-		if err != nil {
+		if os.IsNotExist(err) {
```

Comment:
mm ... if user pass wrong middleware dir should we still error out? Or this is some other cases?

Reply:
in edge there is a lib package